src/main/java/com/celements/cleverreach/Mailing.java

```diff
@@ -10,8 +10,25 @@
   @JsonTypeInfo(use = JsonTypeInfo.Id.NONE, include = As.WRAPPER_OBJECT)
   @JsonSubTypes({ @JsonSubTypes.Type(value = Content.class, name = "content"), })
 
-  public String subject;
-  public Content content;
+  // "name": "my internal name"
```

Comment:
can you please change the commented line to a JavaDoc with an explanaition what the comment is for?